redshift_sqlalchemy/dialect.py

```diff
@@ -19,6 +19,12 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
         __dialect__ = 'redshift'
 
 
+class RedshiftCompiler(PGCompiler):
```

Comment:
is there anything that gets registered to the PGCompiler that sub-classing it might miss?